services/api-server/tests/unit/api_studies/test_api_routes_studies.py

```diff
@@ -133,6 +161,8 @@ async def test_list_study_ports(
     assert resp.status_code == status.HTTP_200_OK
     assert resp.json() == {"items": fake_study_ports, "total": len(fake_study_ports)}
 
+    del respx_mock  # keep alive until server responded
```

Comment:
Why do you need this?

Reply:
I thought it was better then disabling the linting for 'defining unused variables'. Imo this line makes it more clear that we define respx_mock, but don't use it, with the reason being that it shouldn't be destroyed until we want it to.

Reply:
(also, if we have several server call code blocks in a row, it might be good to actively delete the var, so that the unused mock is not there during the next call.

Reply:
I actually saw that when i just do 'not' assign a variable to it, it also works, so i just committed a change without the var name respx_mock being used at all.